languages/ruby/lib/oso/polar/host.rb

```diff
@@ -67,11 +69,12 @@ class Host # rubocop:disable Metrics/ClassLength
       public
 
       attr_writer :accept_expression
-      attr_accessor :build_query, :combine_query, :exec_query
+      attr_accessor :build_query, :combine_query, :exec_query, :adapter
 
       DEFAULT_COMBINE_QUERY = proc { raise 'implement combine_query to use data filtering' }
       DEFAULT_BUILD_QUERY = proc { raise 'implement build_query to use data filtering' }
       DEFAULT_EXEC_QUERY = proc { raise 'implement exec_query to use data filtering' }
+      DEFAULT_ADAPTER = ::Oso::Polar::Data::Adapter.new
```

Comment:
so, this defaults to just one that throws errors? I guess that's cool actually.

Reply:
took this out & made it `nil` to be easier to test if it's configured